cgroup_top.py

```diff
@@ -300,18 +306,23 @@ def collect_ensure_common(data, cgroup):
     data['owner'] = cgroup.owner
     data['type'] = cgroup.type 
 
+def get_user_beacounts():
+    '''
+    get memory stats(via privvmpages) from vzlist output to openvz containers
+    '''
+    prefix = []
+    if os.getuid() != 0:
+       prefix = ['sudo']
+
+    command = prefix + ['vzlist', '-o', 'ctid,privvmpages,privvmpages.l', '-H']
+    return os.popen(" ".join(command)).readlines()
```

Comment:
Yep. This may help: https://docs.python.org/2.6/library/subprocess.html#replacing-os-popen-os-popen2-os-popen3 but using the command list directly without ``shell=True``. Thanks.